src/program/graph.rs

```diff
@@ -240,8 +240,8 @@ impl Default for PreviousNodes {
     /// (in other words, this instruction cannot be scheduled prior to the start of the instruction block).
```

Comment:
This comment refers to blocking but semantically it looks like the block-start is now of an exclusive "using" type; does the comment need revising?

Reply:
Indeed!